source/agora/common/Amount.d

```diff
@@ -260,11 +260,27 @@ public struct Amount
     ***************************************************************************/
 
     pragma(inline, true)
-    public void percentage (ubyte percentage)
+    public bool percentage (ushort pct)
     {
-        assert(percentage <= 100);
-        this.value *= percentage;
+        this.value *= pct;
         this.value /= 100;
+        return isValid();
+    }
+
+    /***************************************************************************
+
+        Take a percentage of the `value` and assert if the result is invalid
+
+        Params:
+            percentage = Requested percentage
+
+    ***************************************************************************/
+
+    pragma(inline, true)
+    public Amount mustPercentage (ushort pct)
```

Comment:
Another name like `getPercentage would be better. And this function is used nowhere.

Reply:
I wanted to be consisten with mustAdd/mustSub